packages/server/graphql/mutations/helpers/bootstrapNewUser.ts

```diff
@@ -94,6 +93,12 @@ const bootstrapNewUser = async (newUser: User, isOrganic: boolean) => {
   }
   analytics.accountCreated(userId, !isOrganic, isPatient0)
 
+  const emailIsVerified = identities[0]?.isEmailVerified
+
+  if (emailIsVerified && isOrganic && tier !== 'enterprise') {
```

Comment:
We decided to not implement this feature for enterprise users. I think at least version 1

Reply:
I only read omitting enterprise SSO users, not enterprise in general. Could you shed some light on why we're skipping enterprise completely?

Reply:
@Dschoordsch Excluding only enterprise with SSO make sense to me. 

In this piece of code only SSO users excluded, because we set `tier = enterprise` when bootstraping a new user only if he is joining using SSO.

However I exclude enterprise completely when searching for matching ors. @acressall Should we allow enterprise without SSO?

I'll merge this PR for now, and make changes in next PRs if needed

Reply:
I think yes, include enterprise except where they have SSO. But I will confirm with Aviva & Trey. I agree, we can merge this PR for now and update if needed